### DISCUSSION TITLE
Trouble installing v2.0.3

### DISCUSSION THREAD

**Discussion by Rumourd** [quietvoid/dovi_tool#229] [General]
I've downloaded this version and extracted to my Windows10 64-bit desktop.  Clicking on DOVI_TOOL does nothing...

For the record, I've been using v.40 for a year now.

**Comment by NeedsMoar** [comment]
It's a command line utility, I'm not sure what your expected behavior is.  Usually command line programs briefly flash a window which closes when they terminate if run from the desktop...  unless you were using v0.4 with a wrapper batch file that prompted for more information I don't recall that dovi_tool itself ever did anything but print help and exit if run with no parameters.

**Comment by Rumourd** [comment]
Yes, v.40 runs a command window in the desktop since I'm not a coder type
(I need some form of GUI).

**Comment by zuba0012** [comment]
This is not a valid issue, as dovi_tool has never functioned this way. You may be referring to a script that involves dovi_tool somehow, and if so, they are not available here.

**Comment by quietvoid** [comment]
Right. I think what you're looking for is DDVT Tool: https://forum.doom9.org/showthread.php?t=183479
I don't provide support for wrapper scripts/GUIs.
